src/components/pages/NotFound/PageNotFound.jsx

```diff
@@ -1,27 +1,25 @@
 import React from "react";
 import { Helmet } from "react-helmet-async";
 import "./PageNotFound.scss";
-import Logo from "../../../utils/rct-logo.png";
 import { useNavigate } from "react-router-dom";
+import strings from "../../../config/strings";
+
 const PageNotFound = () => {
   const navigate = useNavigate();
   return (
     <>
       <Helmet>
         <title>404 Page not found</title>
       </Helmet>
-      <div className="header">
-        <img src={Logo} className="logoImage" alt="Website logo" />
-      </div>
       <div className="pageNotFoundContent">
-        <h1>404 - Page not found.</h1>
-        <h2>Unfortunately, this page doesn't exist.</h2>
+        <h1>404 - Not Found</h1>
```

Comment:
Can we add an icon?

https://mui.com/material-ui/material-icons/?query=glass&selected=SearchOff

Most probably, we can make a ticket with illustrations and ask the team to provide them
In this case, it would be nice to use the logo and play with it, adding a magnifying glass or something, what do you think?

Reply:
Ok, we can add illustration for this page. I think images should be selected on a component basis, maybe we can discuss it on monday.